### DISCUSSION TITLE
Jackett

### DISCUSSION THREAD

**Discussion by nabios2007** [pymedusa/Medusa#10961] [General]
Hi,

I was wondering if Jackett is still supported in Medusa?
I wanted to use it but I don't have the Custom Jacket Providers Tab as described in the Wiki
(https://github.com/pymedusa/Medusa/wiki/Using-Jackett-with-Medusa)

My first goal was to add Torrent9/OxTorrent as a provider (I think it was in the provider list it in the past but not anymore).

Or Should I go for Prowlarr?

**Comment by nabios2007** [comment]
Ok, let's forget this question, I went to Prowlarr

**Comment by nydave69** [comment]
I think the whole purpose of that change was to get users to switch to Prowlarr.
Since that change doesn't make any sense, 
Now one has to maintain the providers in both places.
I first started seeing this push to make this change in Radarr even thou the Jacket info showed the single link.
